src/utils.test.ts

```diff
@@ -99,4 +112,39 @@ describe('words', () => {
     expect(result).toEqual(expected)
     type test = Expect<Equal<typeof result, Mutable<typeof expected>>>
   })
+
+  test('truncate small sentence does nothing', () => {
```

Comment:
The tests here are inside the "words" describe. Could you move it to it's own?

Reply:
fixed in [96f0483](https://github.com/gustavoguichard/string-ts/pull/41/commits/96f048307411594dfc3bb170f23f642f5d519e42)